services/cloudrun-service/Makefile

```diff
@@ -0,0 +1,95 @@
+# This file is part of kuberpult.
+
+# Kuberpult is free software: you can redistribute it and/or modify
+# it under the terms of the Expat(MIT) License as published by
+# the Free Software Foundation.
+
+# Kuberpult is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+# Copyright 2023 freiheit.com
+
+
+include ../../Makefile.variables
+
+MAKEFLAGS += --no-builtin-rules
+
+export CGO_ENABLED=0
+
+IMAGE_REGISTRY?=ghcr.io/freiheit-com/kuberpult
+GO := go
+GO_FILES := $(shell find . -type f -name '*.go')
+PKG_GO_FILES := $(shell find ../../pkg/ -type f -name '*.go')
+ALL_GO_FILES := $(GO_FILES) $(PKG_GO_FILES)
+SERVICE_NAME := $(shell basename $$PWD)
+IMAGENAME?=$(IMAGE_REGISTRY)/kuberpult-cloudrun-service:${VERSION}
+ARTIFACT_REGISTRY_IMAGE_NAME=europe-west3-docker.pkg.dev/fdc-public-docker-registry/kuberpult/kuberpult-cloudrun-service:${VERSION}
+EARTHLY := earthly
+GARBAGE := *.sentinel
+proto:
+	$(MAKE) -C../../pkg/api all
+
+bin/main: $(ALL_GO_FILES) | proto
+	mkdir -p bin
+	$(GO) build -o bin/main ./cmd/server/
+GARBAGE += bin/main
+
+export KUBERPULT_CDSERVER?=localhost:8443
```

Comment:
this looks like it would be long in a docker-compose file, not really in a Makefile

Reply:
This is actually a copy-paste error. I'll remove this.